### PR TITLE
Dockerfile: supporting newrelic

### PR DESCRIPTION
**Pull request by bryanlatten**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -27,17 +31,25 @@ RUN apt-get update && \
         php5-mcrypt \
         php5-json \
         php5-xdebug \
-        nano && \
-    php5dismod xdebug
-
+        newrelic-php5 \
+        nano \
```
Comment (chrishenry) [inline Dockerfile:35]: vim 4 lyfe.

Comment (bryanlatten) [inline Dockerfile:35]: `vi` is already installed, just adding both sides of the spectrum

File: Dockerfile [hunk 2]
```diff
@@ -1,15 +1,19 @@
 FROM behance/docker-nginx:1.1.1
 MAINTAINER Bryan Latten <latten@adobe.com>
 
-# Install singularity_runner
-RUN apt-get install build-essential ruby1.9.1-dev -y && \
```
Comment (bryanlatten) [inline Dockerfile:5]: note: singularity _no longer_ an internal container dependency. only operates from the outside

**Comment by mannytoledo** [comment]
:+1:

File: Dockerfile (hunk 1 above)
Comment (jimdelois) [inline Dockerfile:35]: SO happy to have `nano` back in my life again.

File: Dockerfile [hunk 3]
```diff
@@ -27,17 +31,25 @@ RUN apt-get update && \
...
-
-RUN printf "\n" | pecl install apcu-4.0.7 && \
+    php5enmod igbinary && \
+    printf "\n" | pecl install apcu-4.0.7 && \
     echo 'extension=apcu.so' > /etc/php5/mods-available/apcu.ini && \
     php5enmod apcu
 
+# Prevent newrelic daemon from auto-spawning; uses newrelic run.d script to enable at runtime, when ENV variables are present
+# @see https://docs.newrelic.com/docs/agents/php-agent/advanced-installation/starting-php-daemon-advanced
+RUN sed -i "s/;newrelic.daemon.dont_launch = 0/newrelic.daemon.dont_launch = 3/" /etc/php5/mods-available/newrelic.ini
```
Comment (chrishenry) [inline Dockerfile:52]: Is there a place where you actually start the newrelic daemon?

**Comment by jimdelois** [comment]
:white_check_mark:

File: Dockerfile (hunk 3 above)
Comment (bryanlatten) [inline Dockerfile:52]: newrelic starts automatically with php fpm in this mode (took me awhile to figure out)

**Pull request merged**